packages/admin/src/Pages/Actions/Concerns/HasTooltip.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+namespace Filament\Pages\Actions\Concerns;
+
+use Closure;
+use Illuminate\Support\Str;
+
+trait HasTooltip
+{
+    protected string | Closure | null $tooltip = null;
+
+    public function tooltip(string | Closure | null $tooltip): static
+    {
+        $this->tooltip = $tooltip;
+
+        return $this;
+    }
+
+    public function getTooltip(): ?string
+    {
+        return $this->tooltip;
```

Comment:
This should return `$this->evaluate($this->tooltip)` so that `Closure` arguments get resolved correctly.

Reply:
$this->evaluate is not available here

Reply:
Ah, in which case it should be `return value($this->tooltip)`.